webdriver-spec.html

```diff
@@ -4098,6 +4103,23 @@
 </section> <!-- /Partial Link Text -->
 
 <section>
+<h4>Tag Name</h4>
+
+<p>To find a <a>web element</a> with the <dfn>Tag
+ Name</dfn> <a>strategy</a> the following steps need to be completed:
+
+<ol>
+ <li><p><p>Let <var>elements</var> be the result of
+ calling <a>getElementsByTagName</a> with
+ argument <var>selector</var>, with the <a>context object</a> equal to
+ the <var>start node</var>.
+
+ <li><p>Return <var>elements</var>.
```

Comment:
I don’t think we need the steps here at all: all this does is assign to _elements_, then returning it.

The above can be shortened to:

> Fo find an element with the Tag Name strategy, return the result of calling _start node_’s _getElementsByTagName_ with the argument _selector_.